app/uk/gov/hmrc/submitpublicpensionadjustment/views/PdfSection.scala.xml

```diff
@@ -14,6 +14,10 @@
             </fo:block>
         }
 
-        @section.rows(messages).map { row => @line(row.displayLabel, row.displayValue) }
+        @section.rows(messages).map { row =>
```

Comment:
Should convey how a row should be displayed or not using a more obvious property or just not included in the section rows if it is not to be displayed.